cfn-resources/test/e2e/search-deployment/searchdeployment_template.json

```diff
@@ -0,0 +1,41 @@
+{
+  "AWSTemplateFormatVersion": "2010-09-09",
+  "Description": "This template creates a Search Index on the MongoDB Atlas API, this will be billed to your Atlas account.",
```

Comment:
we can rename this description, can see it was a miss coming from the examples section

Reply:
yes, this is from the examples, do you have any suggestion?

Reply:
thx, updated